views/users-page.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="/public/styles.css" />
```

Comment:
after this general styles linking here, you can also link the css file for the users-page . Will be easier and also help to avoid style clashes in future.